### ISSUE TITLE
Can I use  CompreFace API to detect faces and extract vectors so I can then add then in FAISS to search a lot of faces?

### ISSUE THREAD

**Issue by martinenkoEduard** [exadel-inc/CompreFace#809]
Can I use  CompreFace API to detect faces and extract vectors so I can then add then in FAISS to search a lot of faces?
Is there an API call to get a pure vector?

**Comment by pospielov** [comment]
yes, you can use a detection service and use `face_plugins=calculator` parameter

**Comment by martinenkoEduard** [comment]
What is dimension/size of a vector that it returns?
Have anyone tried to store it in Faiss or milvus?

**Comment by pospielov** [comment]
FaceNet and ArcFace r100 vectors are 512 and the MobileNet ArcFace vector is 128 long.
I don't see any problems why they can't be stored in Faiss or Milvus

**Comment by martinenkoEduard** [comment]
> FaceNet and ArcFace r100 vectors are 512 and the MobileNet ArcFace vector is 128 long. I don't see any problems why they can't be stored in Faiss or Milvus

Thanks!)
